packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -71,5 +73,14 @@ export const comboboxTemplate: FoundationElementTemplate<
                 })}
             ></slot>
         </div>
+        <label
+            part="label"
+            class="${x =>
+                x.slottedLabel && x.slottedLabel.length
+                    ? "label"
+                    : "label label__hidden"}"
```

Comment:
Should there be some styles to go with `label_hidden`?

Reply:
Foundation is unstyled so there won't be exported styles here. With the addition of #6113, we'll have some base styles just to validate, and that work will come when it's merged.

Reply:
Still trying to get used to that...